common/views/components/Modal/Modal.tsx

```diff
@@ -38,12 +55,12 @@ const Overlay = styled.div`
   `}
 `;
 
-const CloseButton = styled(Space).attrs({
+const CloseButton = styled(Space).attrs<HideFocusAttr>({
```

Comment:
Could we stick to the naming of CloseButtonProps if we're going to use it?

Reply:
:+1: